src/lstate.h

```diff
@@ -13,6 +13,10 @@
 #include "ltm.h"
 #include "lzio.h"
 
+#ifdef USE_YK
+#  include <yk.h>
```

Comment:
Is that indentation intentional? I half expect you to tell me that's how they do it :P

Reply:
I maintained the indentation when I moved the code (and, FWIW, I tend to prefer this indentation style, as it makes it easier to see which `#XYZ` code is contained in which `#ifdef` block).